docs/specs/linear-watcher-multiple-repositories/spec.md

```
@@ -0,0 +1,96 @@
+`linear_issue_watches` gains one column; the legacy columns stay and mirror the first entry for downgrade safety.
+
+```
+linear_issue_watches
+  repositories_json  TEXT NOT NULL DEFAULT ''   # JSON array of IssueWatchRepository
+  repository_id      TEXT NOT NULL DEFAULT ''   # legacy — mirrors first entry, read-compat fallback
+  base_branch        TEXT NOT NULL DEFAULT ''   # legacy — mirrors first entry, read-compat fallback
+```
+
+`repositories_json` is the canonical store of the binding. Wire shape of one entry:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)